ansys/mapdl/core/launcher.py

```diff
@@ -387,7 +387,13 @@ def _get_available_base_ansys():
     """
     base_path = None
     if os.name == 'nt':
-        base_path = os.path.join(os.environ['PROGRAMFILES'], 'ANSYS INC')
+        supported_versions = [194, 202, 211, 212]
```

Comment:
At first, I thought we were restricting to only the past four releases, but appears that if it's unsupported, it will just use the same "legacy" behavior.